### PR TITLE
Fixes for docker changes plus others

### PR DESCRIPTION
**Pull request by Frooodle**
# Description
Docker file fixes 
wkhtmltopdf swap to calibre 
Blank page detection using plain java instead of python
Stamp page selection (needs webUI)
PDF to txt readd

TODO before release
stamp page UI
html test
game release fix

Merging early to fix alpha branch
Closes #(issue_number)

## Checklist:

- [x ] I have read the [Contribution Guidelines](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/CONTRIBUTING.md)
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings

## Contributor License Agreement

By submitting this pull request, I acknowledge and agree that my contributions will be included in Stirling-PDF and that they can be relicensed in the future under the MPL 2.0 (Mozilla Public License Version 2.0) license.

(This does not change the general open-source nature of Stirling-PDF, simply moving from one license to another license)

### CODE REVIEW

File: src/main/java/stirling/software/SPDF/controller/api/misc/BlankPageController.java [hunk 1]
```diff
@@ -63,63 +61,35 @@ public ResponseEntity<byte[]> removeBlankPages(@ModelAttribute RemoveBlankPagesR
...
                         // Render image and save as temp file
-                        BufferedImage image = pdfRenderer.renderImageWithDPI(pageIndex, 300);
-                        ImageIO.write(image, "png", tempFile.toFile());
-
-                        List<String> command =
-                                new ArrayList<>(
-                                        Arrays.asList(
-                                                "python",
-                                                System.getProperty("user.dir")
-                                                        + "/scripts/detect-blank-pages.py",
```
Comment (sbplat) [inline src/main/java/stirling/software/SPDF/controller/api/misc/BlankPageController.java:90]: We should delete `detect-blank-pages.py` if it's not used anymore

File: src/main/java/stirling/software/SPDF/controller/api/misc/BlankPageController.java [hunk 2]
```diff
@@ -142,4 +112,30 @@ public ResponseEntity<byte[]> removeBlankPages(@ModelAttribute RemoveBlankPagesR
...
+                    whitePixels++;
+                }
+            }
+        }
+
+        double whitePixelPercentage = (whitePixels / (double) totalPixels) * 100;
+        logger.info(String.format("Page has white pixel percent of %.2f%%", whitePixelPercentage));
+
+        return whitePixelPercentage >= whitePercent;
+    }
```
Comment (sbplat) [inline src/main/java/stirling/software/SPDF/controller/api/misc/BlankPageController.java:140]: Does this close #752?

**Comment by Frooodle** [comment]
@CodiumAI-Agent /improve

**Comment by Frooodle** [comment]
@CodiumAI-Agent /describe

**Comment by Frooodle** [comment]
@CodiumAI-Agent /review

**Pull request merged**